### ISSUE TITLE
DDL: Write keyboard only e2e tests

### ISSUE THREAD

**Issue by sortizsh** [department-of-veterans-affairs/va.gov-team#49849]
## Value Statement

**_As a_** front end developer
**_We want to_** make sure that all the DDL code has e2e testing coverage
**_So that_** we can ensure we're releasing a quality product

## Background Context

DDL is getting closer to release, we need to ensure all important flows have e2e test coverage, including keyboard-only testing

## Tasks
- [ ] Add keyboard-only e2e tests

## Definition of done

- [ ] All tasks complete
- [ ] All tests passing

**Comment by jacobworrell** [comment]
@samcoforma please create separate ticket for work needed to unblock this.

**Comment by HeatherWidmont** [comment]
Sam will come back to this ticket when he has bandwidth in the next couple of days

**Comment by jerekshoe** [comment]
@samcoforma I think we can close this one as well
